test/grpc.js

```diff
@@ -86,31 +87,54 @@ describe('grpc', function() {
       this.address = address;
       this.creds = creds;
       this.options = options;
-    };
-    var grpcClient = gaxGrpc();
+    }
+    var grpcClient;
+    var dummyChannelCreds = {channelCreds: 'dummy'};
+
+    beforeEach(function() {
+      var stubAuth = {getAuthClient: sinon.stub()};
+      var stubGrpc = {credentials: {
+        createSsl: sinon.stub(),
+        combineChannelCredentials: sinon.stub(),
+        createFromGoogleCredential: sinon.stub()
+      }};
+      var dummyAuth = {authData: 'dummy'};
```

Comment:
nit: having all test data use the value `'dummy'` makes it ambiguous which value is going to be used for the channelCreds. It would be good to rename these to `'dummyAuthData'`, `'dummyCreds'`, and `'dummyGrpcAuth'`.